jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/publishers/DependenciesLister.java

```diff
@@ -24,10 +24,10 @@ public class DependenciesLister {
      * @return list of {@link MavenArtifact}
      */
     @Nonnull
-    public static List<MavenDependency> listDependencies(final Element mavenSpyLogs,
+    public static Set<MavenDependency> listDependencies(final Element mavenSpyLogs,
```

Comment:
this has broken backward compat!

Reply:
Changing List to Set is not backward compatible.
Yes please revert it

Reply:
here is a stacktrace @bguerin showing the issue : https://ci-maven.apache.org/job/Maven/job/maven-box/job/maven-site/job/master/188/consoleFull
This code is a public API used by some extensions AFAIU

Reply:
thanks @aheritier this is what I missed

@olamy I will revert the public signature change, and release it as 3.11.1

Reply:
@olamy done

https://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/pipeline-maven/3.11.1/pipeline-maven-3.11.1.jar

Reply:
@bguerin Thanks

Reply:
@bguerin would be good to have a release of master as well because users using last release such `1235.v2db_ddd9f797b`  will never see 3.11.1 as an upgrade